app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def channels
+
+    if(params[:uuid] == nil)
```

Comment:
I'd prefer to use `.blank?` or `.nil?` it's more ruby-esque. You can view the [differences here](https://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails)

Reply:
Agree it looks nicer to use `.nil?`, Fixed.